sunpy/map/mapbase.py

```diff
@@ -1559,66 +1556,65 @@ def draw_limb(self, axes=None, **kwargs):
 
         return [circ]
 
+    @deprecate_positional_args
     @u.quantity_input
-    def draw_rectangle(self, bottom_left, width: u.deg, height: u.deg, axes=None, **kwargs):
+    def draw_rectangle(self, bottom_left, *, width: u.deg=None, height: u.deg=None, axes=None, top_right=None, **kwargs):
```

Comment:
The fact we are changing `axes` to a keyword only argument here is an interesting side effect. I can't see an obvious way around it.